docs/tutorials/gsoc-ideas-list.md

```diff
@@ -39,8 +39,9 @@ These projects are intended to be good starting projects for new Shaka Player
 contributors, to complete relatively independently in a few weeks. Familiarity 
 with Javascript or Python will be helpful, but not required.
 
-To start, checkout our [demo][] and [tutorial][] page, and
```

Comment:
This doc will be deleted soon. Would you please drop the changes in this file? Thank you!

Reply:
Yes, mam.